### ISSUE TITLE
Populate ticket department from email field

### ISSUE THREAD

**Issue by Luc-IT** [osTicket/osTicket#3713]
### Prerequisites

* [ Y] Can you reproduce the problem in a fresh installation of the "develop" branch?
* [ N] Do you have any errors in the PHP error log, or javascript console?
* [ Y] Did you check the [osTicket forums](http://osticket.com/forum/categories)?
* [ Y] Did you [perform a cursory search](https://github.com/osTicket/osTicket/issues) to see if your bug or enhancement is already reported?

### Description

Hi,
we are testing tickets creating from email POP. It does work.
Now, we would like to find a way to populate the ticket's department from parsing a field in the mail body.
Any idea ?
Thanks in advance for your help,

Luc

### Versions
version d'osTicket	v1.9.5.1 
Logiciel serveur Web	Apache/2.2.22 (Debian)
Version de MySQL	5.5.37
Version PHP	5.4.36-0+deb7u3

**Comment by ntozier** [comment]
Upgrade to 1.10.
Create a ticket filter.
for the rule select: Ticket / Issue Details, contains, your unique pattern.

**Comment by Luc-IT** [comment]
Hi,
thanks for your answer.
We'll try this upgrade and its filters.
Luc

**Comment by txzsfj** [comment]
I have been trying to get this to work, with absolutely no luck (running 1.10). Here's what I've done so far:

1) Created a new Help Topic named "Email Ticket." Set to private, as it shouldn't be seen on web form.
2) For the default email address, which is piping just fine, all inbounds are assigned to Email Ticket.
3) Set up a new filter as a test (I'll need many similar ones, if I get this to work. The filter looks like this:

Filter Name:	  Email Ticket - TestPhrase
Execution Order:	 (1...99)  1   
Filter Status:	 Active
Target Channel:	   Emails

(For the Target Channel, I have also tried Any and the email address in the System Calls section.)

For the Filter Rules, I have two, set to both must be true:

1) Ticket/Issue Details - contains - TestPhrase (have also tried *TestPhrase*)
2) Topic ID - equals - 32 (the topic ID of the Email Ticket help topic; also tried its name)

For the Filter Action, there's just one:

Set Help Topic: [Some other help topic]

However, it simply will not process inbound emails. They get the proper Email Ticket department, but they do not move to the Help Topic I set in the rules.

There's nothing in the log files, and no way to tell what's going on; it's a black hole, and I'm stumped. All ideas appreciated.

-r
